pkg/tsdb/testdatasource/scenarios.go

```diff
@@ -47,7 +47,7 @@ func init() {
 			factor := 2
 			for i := 0; i < 10; i++ {
 				timeWalkerMs := context.TimeRange.GetFromAsMsEpoch()
-				serie := &tsdb.TimeSeries{Name: strconv.Itoa(start)}
+				timeSeries := &tsdb.TimeSeries{Name: strconv.Itoa(start)}
```

Comment:
```suggestion
				series := &tsdb.TimeSeries{Name: strconv.Itoa(start)}
```

Reply:
I can't do this, there's a series variable on the next line. I hit that in CI as well.